app/src/androidTest/java/mx/yellowme/youst/util/DataBindingIdlingResource.kt

```diff
@@ -8,7 +8,7 @@ import androidx.fragment.app.FragmentActivity
 import androidx.fragment.app.testing.FragmentScenario
 import androidx.test.core.app.ActivityScenario
 import androidx.test.espresso.IdlingResource
-import java.util.UUID
+import java.util.*
```

Comment:
Is it necessary to use the`*` import? I think we are just using the `UUID`

Reply:
Not sure, let me check 😬